pkgs/applications/video/aegisub/default.nix

```diff
@@ -60,58 +70,84 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "wangqr";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-Er0g8fJyx7zjNVpKw7zUHE40hU10BdYlZohlqJq2LE0=";
+    rev = "91f8b5f91eb960bad19899c10af08aca34f9b697";
```

Comment:
If you are using a raw Git hash revision, then you should edit `version` accordingly.

Reply:
Changed to 3.3.2-20220612.